ddtrace/contrib/django/utils.py

```diff
@@ -292,6 +292,10 @@ def _after_request_tags(pin, span, request, response):
             # Note: getattr calls to user / user_is_authenticated may result in ImproperlyConfigured exceptions from
             # Django's get_user_model():
             # https://github.com/django/django/blob/a464ead29db8bf6a27a5291cad9eb3f0f3f0472b/django/contrib/auth/__init__.py
+            #
+            # FIXME: getattr calls to user fail in async contexts.
```

Comment:
Would using `object.__getattribute__` solve this issue entirely?

Reply:
I don't think so, it seems like the User attributes are delay fetched and fetched synchronously.

What we probably need to do is get an example app together for Django asgi/async that causes this same error and see how we should fetch this user data that is safe for async context.